### ISSUE TITLE
[WCAG 2.2 AA  ACCESSIBILITY]    SIGN-UP PAGE

### ISSUE THREAD

**Issue by curtis-taylor** [torontojs/volunteer#167]
**_ANALYSIS REPORT created by Axe-Core/Playwright module for level WCAG 2.2 AA._**

**URL:** _http://localhost:3000/pages/sign-up/_

# Instructions

- Following Playwright test failed.
- Explain why, be concise, respect Playwright best practices.
- Provide a snippet of code with the fix, if possible.
# Test info

- Name: ASSESSIBILITY Suite >> BASIC WCAG22AA
- Location: C:\Users\curti\VMS_FE_Testing\Playwright_FE_Tests\tests\sign-up.spec.ts:478:9

# Error details

```

- Expected  -   1
+ Received  + 238

- Array []
+ Array [
+   Object {
+     "description": "Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds",
+     "help": "Elements must meet minimum color contrast ratio thresholds",
+     "helpUrl": "https://dequeuniversity.com/rules/axe/4.10/color-contrast?application=playwright",
+     "id": "color-contrast",
+     "impact": "serious",
+     "nodes": Array [
+       Object {
+         "all": Array [],
+         "any": Array [
+           Object {
+             "data": Object {
+               "bgColor": "#e5e5e5",
+               "contrastRatio": 2.26,
+               "expectedContrastRatio": "4.5:1",
+               "fgColor": "#999999",
+               "fontSize": "10.0pt (13.3333px)",
+               "fontWeight": "normal",
+               "messageKey": null,
+             },
+             "id": "color-contrast",
+             "impact": "serious",
+             "message": "Element has insufficient color contrast of 2.26 (foreground color: #999999, background color: #e5e5e5, font size: 10.0pt (13.3333px), font weight: normal). Expected contrast ratio of 4.5:1",
+             "relatedNodes": Array [
+               Object {
+                 "html": "<input id=\"name-input\" type=\"text\" name=\"name\" placeholder=\"Your name\" minlength=\"1\" required=\"\" aria-describedby=\"name-input-helper-text\">",
+                 "target": Array [
+                   "#name-input",
+                 ],
+               },
+             ],
+           },
+         ],
+         "failureSummary": "Fix any of the following:
+   Element has insufficient color contrast of 2.26 (foreground color: #999999, background color: #e5e5e5, font size: 10.0pt (13.3333px), font weight: normal). Expected contrast ratio of 4.5:1",
+         "html": "<input id=\"name-input\" type=\"text\" name=\"name\" placeholder=\"Your name\" minlength=\"1\" required=\"\" aria-describedby=\"name-input-helper-text\">",
+         "impact": "serious",
+         "none": Array [],
+         "target": Array [
+           "#name-input",
+         ],
+       },
+       Object {
+         "all": Array [],
+         "any": Array [
+           Object {
+             "data": Object {
+               "bgColor": "#ffffff",
+               "contrastRatio": 3.99,
+               "expectedContrastRatio": "4.5:1",
+               "fgColor": "#ff0000",
+               "fontSize": "9.0pt (12px)",
+               "fontWeight": "normal",
+               "messageKey": null,
+             },
+             "id": "color-contrast",
+             "impact": "serious",
+             "message": "Element has insufficient color contrast of 3.99 (foreground color: #ff0000, background color: #ffffff, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1",
+             "relatedNodes": Array [],
+           },
+         ],
+         "failureSummary": "Fix any of the following:
+   Element has insufficient color contrast of 3.99 (foreground color: #ff0000, background color: #ffffff, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1",
+         "html": "<span>REQUIRED</span>",
+         "impact": "serious",
+         "none": Array [],
+         "target": Array [
+           "label[for=\"email-input\"] > span",
+         ],
+       },
+       Object {
+         "all": Array [],
+         "any": Array [
+           Object {
+             "data": Object {
+               "bgColor": "#e5e5e5",
+               "contrastRatio": 2.26,
+               "expectedContrastRatio": "4.5:1",
+               "fgColor": "#999999",
+               "fontSize": "10.0pt (13.3333px)",
+               "fontWeight": "normal",
+               "messageKey": null,
+             },
+             "id": "color-contrast",
+             "impact": "serious",
+             "message": "Element has insufficient color contrast of 2.26 (foreground color: #999999, background color: #e5e5e5, font size: 10.0pt (13.3333px), font weight: normal). Expected contrast ratio of 4.5:1",
+             "relatedNodes": Array [
+               Object {
+                 "html": "<input id=\"email-input\" type=\"email\" name=\"email\" placeholder=\"Your account e-mail\" required=\"\" aria-describedby=\"email-input-helper-text\">",
+                 "target": Array [
+                   "#email-input",
+                 ],
+               },
+             ],
+           },
+         ],
+         "failureSummary": "Fix any of the following:
+   Element has insufficient color contrast of 2.26 (foreground color: #999999, background color: #e5e5e5, font size: 10.0pt (13.3333px), font weight: normal). Expected contrast ratio of 4.5:1",
+         "html": "<input id=\"email-input\" type=\"email\" name=\"email\" placeholder=\"Your account e-mail\" required=\"\" aria-describedby=\"email-input-helper-text\">",
+         "impact": "serious",
+         "none": Array [],
+         "target": Array [
+           "#email-input",
+         ],
+       },
+       Object {
+         "all": Array [],
+         "any": Array [
+           Object {
+             "data": Object {
+               "bgColor": "#ffffff",
+               "contrastRatio": 3.99,
+               "expectedContrastRatio": "4.5:1",
+               "fgColor": "#ff0000",
+               "fontSize": "9.0pt (12px)",
+               "fontWeight": "normal",
+               "messageKey": null,
+             },
+             "id": "color-contrast",
+             "impact": "serious",
+             "message": "Element has insufficient color contrast of 3.99 (foreground color: #ff0000, background color: #ffffff, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1",
+             "relatedNodes": Array [],
+           },
+         ],
+         "failureSummary": "Fix any of the following:
+   Element has insufficient color contrast of 3.99 (foreground color: #ff0000, background color: #ffffff, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1",
+         "html": "<span>REQUIRED</span>",
+         "impact": "serious",
+         "none": Array [],
+         "target": Array [
+           "label[for=\"password-input\"] > span",
+         ],
+       },
+       Object {
+         "all": Array [],
+         "any": Array [
+           Object {
+             "data": Object {
+               "bgColor": "#e5e5e5",
+               "contrastRatio": 2.26,
+               "expectedContrastRatio": "4.5:1",
+               "fgColor": "#999999",
+               "fontSize": "10.0pt (13.3333px)",
+               "fontWeight": "normal",
+               "messageKey": null,
+             },
+             "id": "color-contrast",
+             "impact": "serious",
+             "message": "Element has insufficient color contrast of 2.26 (foreground color: #999999, background color: #e5e5e5, font size: 10.0pt (13.3333px), font weight: normal). Expected contrast ratio of 4.5:1",
+             "relatedNodes": Array [
+               Object {
+                 "html": "<input id=\"password-input\" type=\"password\" name=\"password\" placeholder=\"Your password\" required=\"\" aria-describedby=\"password-input-strength password-input-suggestion\">",
+                 "target": Array [
+                   "#password-input",
+                 ],
+               },
+             ],
+           },
+         ],
+         "failureSummary": "Fix any of the following:
+   Element has insufficient color contrast of 2.26 (foreground color: #999999, background color: #e5e5e5, font size: 10.0pt (13.3333px), font weight: normal). Expected contrast ratio of 4.5:1",
+         "html": "<input id=\"password-input\" type=\"password\" name=\"password\" placeholder=\"Your password\" required=\"\" aria-describedby=\"password-input-strength password-input-suggestion\">",
+         "impact": "serious",
+         "none": Array [],
+         "target": Array [
+           "#password-input",
+         ],
+       },
+       Object {
+         "all": Array [],
+         "any": Array [
+           Object {
+             "data": Object {
+               "bgColor": "#ffffff",
+               "contrastRatio": 4.04,
+               "expectedContrastRatio": "4.5:1",
+               "fgColor": "#ed3731",
+               "fontSize": "13.5pt (18px)",
+               "fontWeight": "normal",
+               "messageKey": null,
+             },
+             "id": "color-contrast",
+             "impact": "serious",
+             "message": "Element has insufficient color contrast of 4.04 (foreground color: #ed3731, background color: #ffffff, font size: 13.5pt (18px), font weight: normal). Expected contrast ratio of 4.5:1",
+             "relatedNodes": Array [],
+           },
+         ],
+         "failureSummary": "Fix any of the following:
+   Element has insufficient color contrast of 4.04 (foreground color: #ed3731, background color: #ffffff, font size: 13.5pt (18px), font weight: normal). Expected contrast ratio of 4.5:1",
+         "html": "<a href=\"/pages/sign-in\" class=\"underline\">Click here to log-in</a>",
+         "impact": "serious",
+         "none": Array [],
+         "target": Array [
+           ".underline",
+         ],
+       },
+       Object {
+         "all": Array [],
+         "any": Array [
+           Object {
+             "data": Object {
+               "bgColor": "#ffffff",
+               "contrastRatio": 2.84,
+               "expectedContrastRatio": "4.5:1",
+               "fgColor": "#999999",
+               "fontSize": "12.7pt (16.8697px)",
+               "fontWeight": "bold",
+               "messageKey": null,
+             },
+             "id": "color-contrast",
+             "impact": "serious",
+             "message": "Element has insufficient color contrast of 2.84 (foreground color: #999999, background color: #ffffff, font size: 12.7pt (16.8697px), font weight: bold). Expected contrast ratio of 4.5:1",
+             "relatedNodes": Array [],
+           },
+         ],
+         "failureSummary": "Fix any of the following:
+   Element has insufficient color contrast of 2.84 (foreground color: #999999, background color: #ffffff, font size: 12.7pt (16.8697px), font weight: bold). Expected contrast ratio of 4.5:1",
+         "html": "<p class=\"text-helper\">TorontoJS</p>",
+         "impact": "serious",
+         "none": Array [],
+         "target": Array [
+           ".text-helper",
+         ],
+       },
+     ],
+     "tags": Array [
+       "cat.color",
+       "wcag2aa",
+       "wcag143",
+       "TTv5",
+       "TT13.c",
+       "EN-301-549",
+       "EN-9.1.4.3",
+       "ACT",
+     ],
+   },
+ ]

**Comment by curtis-taylor** [comment]
After retest. Issue is resolved.
